### ISSUE TITLE
possibility to attach photos to notes

### ISSUE THREAD

**Issue by tyrasd** [openstreetmap/openstreetmap-website#395]
Sometimes when adding a note it is not so easy to explain the actual map bug just with words alone (let's say you find a very complicated road sign and don't really know what's going on). In that case it would be helpful for both the note submitter and the mappers if photos of the location could be attached to the note.

**Comment by westnordost** [comment]
In other words, would you accept a PR if someone was to implement it?

**Comment by tomhughes** [comment]
Only if OWG were prepared to provide significant server resources to support the service.

**Comment by 1ec5** [comment]
Photos could provide useful context in a note, but allowing images in notes could also increase the potential for abuse. Inappropriate images including copyright infringement could become more common.

**Comment by dankarran** [comment]
This would be a handy feature, though as others have noted it's potentially open for misuse.

From a quick read through https://github.com/openstreetmap/operations/issues/206 and https://github.com/openstreetmap/operations/issues/169 it looks like S3 may now be in place as an object store - does that mean the storage blocker for this has gone?

**Comment by gravitystorm** [comment]
Yes, there's two technical aspects that have changed since these related tickets were opened. The first is that OSMF is using an object store for hosting files (e.g. user avatars and gpx traces), currently AWS S3, and secondly Rails has the ActiveStorage feature which takes care of the uploading / downloading / image resizing etc and makes it much easier (less code) to add this kind of functionality.

The policy topics - e.g. inappropriate image hosting, moderation etc - are unaffected and there hasn't been any recent discussions on this topic as far as I'm aware.

**Comment by tomhughes** [comment]
Well there are budgetary questions if we're going to wind up using substantially more space and image hosting is also potentially problematic from a moderation and legal liability point of view so we may want to speak to LWG before considering it.
